src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -12,6 +12,7 @@
     public static final String MESSAGE_MODULES_LISTED = "%1$d modules listed!";
     public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "This module does not exist in FaculType";
     public static final String MESSAGE_DUPLICATE_MODULE = "This module already exists in FaculType";
+    public static final String MESSAGE_INSTRUCTOR_DOES_NOT_EXIST = "This instructor does not exist in this module.";
```

Comment:
Does "This instructor is not assigned to this module" sound better? Also no need to use a dot.

Reply:
Got it. Will change it.